e2e/cypress/integration/channel/header_spec.js

```diff
@@ -53,7 +57,7 @@ describe('Header', () => {
         // # Open a DM with user named 'user-2'
         cy.get('#addDirectChannel').click().wait(TIMEOUTS.TINY);
         cy.focused().type('user-2', {force: true}).type('{enter}', {force: true}).wait(TIMEOUTS.TINY);
-        cy.get('#saveItems').click();
+        cy.get('#saveItems').click().wait(TIMEOUTS.TINY);
```

Comment:
Not related to guest account but this is a bit flaky.  So buy some time before initiating the next step.